client/src/legacy/TinyMCE_sslink-internal.js

```diff
@@ -86,6 +83,20 @@ jQuery.entwine('ss', ($) => {
     },
 
     /**
+     * Determine whether to show the link text field
+     *
+     * @return {Boolean}
+     */
+    getRequireLinkText() {
+      const selection = tinymce.activeEditor.selection;
```

Comment:
noticed this bug here
I'll rebase and correct this too - should be `this.getElement().getEditor().getInstance().selection`